certif/tests/integration/components/team/invitations-list_test.js

```diff
@@ -13,6 +13,30 @@ module('Integration | Component |  team/invitation-list', function (hooks) {
     store = this.owner.lookup('service:store');
   });
 
+  test('displays email address, last sending date and actions headers', async function (assert) {
+    // given
+    this.set('invitations', []);
+
+    // when
+    const screen = await renderScreen(hbs`<Team::InvitationsList @invitations={{this.invitations}} />`);
+
+    // then
+    assert.strictEqual(screen.getAllByRole('columnheader').length, 3);
+    assert
+      .dom(screen.getByRole('columnheader', { name: this.intl.t('pages.team-invitations.table.labels.email-address') }))
+      .exists();
```

Comment:
Dans les tests d'integration coté certif, on test la chaine traduite directement:
```suggestion
    assert
      .dom(screen.getByRole('columnheader', { name: 'Le super nom') }))
      .exists();
```